go.mod

```diff
@@ -0,0 +1,83 @@
+module sigs.k8s.io/secrets-store-sync-controller
+
+go 1.22
+
+require (
+	github.com/google/go-cmp v0.6.0
+	github.com/prometheus/client_golang v1.16.0
+	go.opentelemetry.io/otel/exporters/prometheus v0.38.1
+	go.opentelemetry.io/otel/metric v0.38.1
+	go.opentelemetry.io/otel/sdk/metric v0.38.1
+	golang.org/x/crypto v0.17.0
```

Comment:
Bump this to latest version as old versions have CVE

Reply:
https://github.com/kubernetes-sigs/secrets-store-sync-controller/issues/19 will help surface these before the release.